From 5bca90cbc770d232fe00fbe70b450cb2b596f26b Mon Sep 17 00:00:00 2001
From: "jie.chen" <jie.a.chen@intel.com>
Date: Fri, 28 Feb 2014 05:13:27 +0800
Subject: [PATCH 1/2] fix the __unused issue

__unused is defined as pseudo-keyword in bionic, so can't be used
as variable name
---
 linux_syscall_support.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/linux_syscall_support.h b/linux_syscall_support.h
index d04901d..84cfc81 100644
--- a/linux_syscall_support.h
+++ b/linux_syscall_support.h
@@ -426,7 +426,7 @@ struct kernel_stat {
   uint64_t           st_mtime_nsec_;
   uint64_t           st_ctime_;
   uint64_t           st_ctime_nsec_;
-  int64_t            __unused[3];
+  int64_t            dummy[3];
 };
 #elif defined(__PPC__)
 struct kernel_stat {
-- 
1.8.3.2

